components/infobox/wikis/arenaofvalor/infobox_league.lua

```diff
@@ -0,0 +1,161 @@
+---
+-- @Liquipedia
+-- wiki=arenaofvalor
+-- page=Module:Infobox/League/Custom
+--
+-- Please see https://github.com/Liquipedia/Lua-Modules to contribute
+--
+
+local League = require('Module:Infobox/League')
+local String = require('Module:StringUtils')
+local Variables = require('Module:Variables')
+local Tier = require('Module:Tier')
+local PageLink = require('Module:Page')
+local Class = require('Module:Class')
+local Injector = require('Module:Infobox/Widget/Injector')
+local Cell = require('Module:Infobox/Widget/Cell')
+local Title = require('Module:Infobox/Widget/Title')
+local PrizePoolCurrency = require('Module:Prize pool currency')
+
+local CustomLeague = Class.new()
+local CustomInjector = Class.new(Injector)
+
+local _args
+local _game
+
+local _TODAY = os.date('%Y-%m-%d', os.time())
+local _GAME = mw.loadData('Module:GameVersion')
+
+function CustomLeague.run(frame)
+	local league = League(frame)
+	_args = league.args
+
+	league.addToLpdb = CustomLeague.addToLpdb
+	league.createWidgetInjector = CustomLeague.createWidgetInjector
+	league.defineCustomPageVariables = CustomLeague.defineCustomPageVariables
+
+	return league:createInfobox(frame)
+end
+
+function CustomLeague:createWidgetInjector()
+	return CustomInjector()
+end
+
+function CustomInjector:parse(id, widgets)
+	if id == 'gamesettings' then
+		return {
+			Cell{name = 'Game version', content = {
+					CustomLeague._getGameVersion()
+				}},
+			}
+	elseif id == 'prizepool' then
+		return {
+			Cell{
+				name = 'Prize pool',
+				content = {CustomLeague:_createPrizepool()}
+			},
+		}
+	elseif id == 'liquipediatier' then
+		return {
+			Cell{
+				name = 'Liquipedia tier',
+				content = {CustomLeague:_createTierDisplay()},
+				classes = {_args['moonton-sponsored'] == 'true' and 'valvepremier-highlighted' or ''},
+			},
+		}
+	elseif id == 'customcontent' then
+		if _args.player_number then
+			table.insert(widgets, Title{name = 'Players'})
+			table.insert(widgets, Cell{name = 'Number of players', content = {_args.player_number}})
+		end
+
+		--teams section
+		if _args.team_number or (not String.isEmpty(_args.team1)) then
```

Comment:
what's `|team1=`

Reply:
In the past there were a section where it can list top 4 (below team/player numbers) but those arent used anymore so I'm adjusting this in the next update

Reply:
`|teamX=` is used by some wikis to add a "section" in the infobox where participating teams are listed, it is mostly a legacy thing

if the wiki in question doesn't have it remove it from the module, for new wikis def. remove it